react-custom-hooks/client/src/App.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import { useState } from 'react';
+import PageTitle from './components/PageTitle';
+import TodoList from './components/TodoList';
+import TodoForm from './components/TodoForm';
+import { useTodos } from './components/useTodos';
+
+export default function App() {
+  // TODO: Use the custom hook here instead of state and functions
+  const [error, setError] = useState();
+
+  const useTodo = useTodos();
```

Comment:
It works but the code would be cleaner if you destructured just the fields you are going to use.